config/forms/2021_2022.json

```diff
@@ -1134,10 +1134,25 @@
               "header": "",
               "description": "",
               "questions": {
+                "age1_known": {
+                  "header": "Do you know the lead tenant’s age?",
```

Comment:
I think the first question should be `How many people live in the household for this letting?` ?